sys/include/riotboot/flashwrite.h

```diff
@@ -128,6 +128,11 @@ typedef struct {
  */
 #define RIOTBOOT_FLASHWRITE_SKIPLEN     sizeof(RIOTBOOT_MAGIC)
 
+/**
+ * @brief Magic number used to invalidate a slot
+ */
+#define INVALIDATE_HDR                  0
```

Comment:
This would depend on the flash, in some cases `0x00` is the erase state in others `0xFF`, and therefore this could be a `NOP` depending on the operation.

Reply:
Maybe we should add a `FLASH_ERASE_STATE` value, and use that as the write value. What do you think @bergzand should we add something like that?

Reply:
we could also use `0xAA`, that should always clear half the bits of the signature.